### PR TITLE
 npm cleanup

### PR DESCRIPTION
**Pull request by stevector**
cleaning up some npm dependencies

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -235,14 +235,6 @@ module.exports = {
     },
     `gatsby-transformer-sharp`,
     `gatsby-plugin-sharp`,
-    {
-      resolve: "gatsby-plugin-web-font-loader",
```
Comment (stevector) [inline gatsby-config.js:239]: It appears we weren't using this font at all.

**Comment by stevector** [comment]
@rachelwhitton and @jazzsequence this one is not urgent, but I cleaned up some of our NPM dependencies, mostly by removing packages. I tracked the operations I did in [cleanup.txt](https://github.com/pantheon-systems/documentation/pull/9293/files#diff-043616eca9b10a557d218738b0811410e1324d254ea1b4f7be889ffda255ce65)

File: cleanup.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+npm uninstall algoliasearch
```
Comment (stevector) [inline cleanup.txt:1]: I should delete this file before merging

**Review by rachelwhitton — APPROVED** [review]
I ran local VRT with passing results, looks good to me 👍 

screencapture-localhost-9323-2024-11-05-10_22_07 (1)

**Review by jazzsequence — APPROVED** [review]
this isn't updating anything, it's just removing unused things?

**Comment by stevector** [comment]
@jazzsequence it also contained some updates triggered by `npm update msgpackr` which I ran to fix a single notice in `npm audit`. Then `npm audit fix` did some other additional updates. Resolving most of the warning seems like it would require a major version update to Gatsby which I don't think is worth doing.

**Pull request merged**